src/Components/NotificationDropdown.js

```diff
@@ -0,0 +1,108 @@
+        case 'EVENT_UPDATE':
+            return <i class="fas fa-calendar-day fa-2x" style={{color: isread}}></i>;   
+        case 'PEOPLE_MGMT':
+            return <i class="fas fa-users fa-2x" style={{color: isread}}></i>;
+        default:
+            return "";
+    }
+}
+
+class Notification extends Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
+            allRead: false,
+            notifications: []
+        };
+        this.isAllRead = this.isAllRead.bind(this);
+        this.isRead = this.isRead.bind(this);
+    }
+
+    componentDidMount() {
+        this.getNotifications();
+    }
+
+    isAllRead(){
+        let read = this.state.notifications.slice(0, 10).filter(e => e.is_read == false).length > 0 ? false : true;
+        if ( read != this.state.allRead){
+            this.setState({
+                allRead: !this.state.allRead
+              });
+        }
+    }
+
+    getNotifications() {
+        axios({
+            method: 'get',
+            url: "http://localhost:8080/notifications",
+            params: {
+              pageId: 0,
+              limit: 10,
+            },
+            headers: {
+              'Content-Type': 'application/json',
+              'token': ""
+            },
+          })
+          .then(res => {
+              if(res.status == 200) {
+                  let notifs = [];
+                  res.data.forEach(notification => {
+                      notifs.push(notification);
+                  });
+                  this.setState({notifications: notifs});
+              }
+          })
+    }
+
+    isRead(id, event_id) {
+        axios.post(`http://localhost:8080/notification/${id}/read`, {
+            token: ""
+        });
+        
+        // Redirect to event page
+        // this.props.history.push('/events/' + event_id);
+    }
+
+    render(){
+        this.isAllRead();
+        const notification = this.state.notifications;
+        let notif = notification.slice(0, 10).map((notification, i) => {
+            return  <MDBDropdownItem onClick={() => this.isRead(notification.id, notification.event_id)} style={notification.is_read ? {fontWeight: '400'} : {fontWeight: '900'}}> 
+                        <div className="contents">
+                            <NotifIcon value={notification}/>
+                            <div className="description">{notification.description}</div>
+                        </div>
+                        <MDBDropdownItem divider />
+                    </MDBDropdownItem>
+        });
+
+        return (
+            <Fragment>
+            <MDBDropdown>
+                <MDBDropdownToggle caret color="primary" className="notifdropdown" style={this.state.allRead ? {backgroundColor : 'yellow'} : {backgroundColor: 'yellow'}}>
+                    {this.state.allRead ? <i class="far fa-bell fa-2x"></i> : <i class="fas fa-bell fa-2x" style={{color: "var(--white)"}}></i>}
```

Comment:
I would put the ternary strictly for the style and not for the entire component to be cleaner and more consistent with above.

Reply:
Do I do a switch for this too then? This only toggles between if all notifications are read or not so i thought a ternary would be appropriate

Reply:
No you can use a ternary, I just think it would be cleaner if you did it only for the style like what you did above: 
````
<MDBDropdownItem onClick={() => this.handleRead(notification.id, notification.event_id)} style={notification.is_read ? {fontWeight: '400'} : {fontWeight: '900'}}>
````
rather than for the entire element.